python/tank/util/login.py

```diff
@@ -65,35 +68,47 @@ def get_shotgun_user(sg):
         g_shotgun_user_cache = sg.find_one("HumanUser", filters=[["login", "is", local_login]], fields=fields)
     
     return g_shotgun_user_cache
-        
+
 def get_current_user(tk):
     """
-    Retrieves the current user as a dictionary of metadata values.
-    Returns None if the user is not found in shotgun.
-    
-    Returns the following fields:
-    
-    * id
-    * type
-    * email
-    * login
-    * name
-    * image url (thumbnail)
-    
-    This method connects to shotgun.
-    """    
+    Retrieves the current user as a dictionary of metadata values. Note: This method connects to
```

Comment:
I think the dosctring should probably mention something about the hook:

"If a user has been authenticated via a login prompt, this method will return the credentials associated with that user. If toolkit has been configured to use a script user to connect to shotgun, a core hook will be executed to established which user is associated with the current session. (This is usually based on the currently logged in user)."